using-mobile-extensions/mobile-core/identity/identity-event-reference.md

```diff
@@ -2,68 +2,248 @@
+| `visitoridslist`        | List<Map<String, Object>> | Yes          | A list of visitor identifiers that were previously synced using `syncIdentifier` or `syncIdentifiers` public APIs. Each visitor id will have the following keys: `id_origin`, `id_type`, `id`, `authentication_state` |
+| `lastsync`              | Long                      | Yes          | Timestamp in seconds of the last sync call sent to the Experience Cloud ID Service, by default it is 0 |
+| `updatedurl`            | String                    | Yes          | The update URL when `appendVisitorInfoForURL` (Android) / `appendToUrl` (iOS) public API is called |
+| `urlvariables`          | String                    | Yes          | The Visitor IDs as query parameters string when `getUrlVariables` API is called |
+
+#### Event data example
+
+When all identifiers, blob and location hint are populated
+
+```json
+{
+    "mid": "midExample",
+    "advertisingidentifier": "advertisingIdExample",
+    "pushidentifier": "pushIdExample",
+    "blob": "blobExample",
+    "locationhint": "locationHintExample",
+    "visitoridslist": [{
+        "id_type": "type1",
+        "id_origin": "origin1",
+        "id": "id1",
+        "authentication_state": 1
+    },
+    {
+        "id_type": "type2",
+        "id_origin": "origin2",
+        "id": "id2",
+        "authentication_state": 2
+    }]
+}
+```
+
+#### Event data example
+
+The result for the appendToUrl request
+
+```json
+{
+    "updatedurl": "https://example.com/path?query=parameter1&other=parameter2&adobe_mc=TS%3D1563845864296%7CMCMID%3Dmidexample%7CMCORGID%3Dorgidexample%40AdobeOrg%7CMCAID%3Daidexample&adobe_aa_vid=videxample"
+}
+```
+
+#### Event data example
+
+The result for the getUrlVariables request
+
+```json
+{
+    "urlvariables": "adobe_mc=TS%3D1563845864296%7CMCMID%3Dmidexample%7CMCORGID%3Dorgidexample%40AdobeOrg%7CMCAID%3Daidexample&adobe_aa_vid=videxample"
+}
+```
 
 {% hint style="warning" %}
-The Analytics Custom Visitor ID \(VID\) is no longer included in the data payload. If needed, it may be retrieved from the [Analytics extension](https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/tree/174e9069bc1d3a521b59d52a066e9a7730f60ff5/using-mobile-extensions/adobe-analytics/analytics-api-reference/README.md#get-the-custom-visitor-identifier).
+
+The Analytics Custom Visitor ID \(VID\) is no longer included in the Identity event data payload. If needed, it may be retrieved from the [Analytics extension](/using-mobile-extensions/adobe-analytics/analytics-api-reference/README.md#get-the-custom-visitor-identifier).
+
 {% endhint %}
 
+### Configuration content request 
+
+The Identity Extension dispatches a new Configuration Content Request event when a privacy change is detected in the response received from the Experience Cloud ID Service. This event will be handled by the Core extension and the new privacy status will be broadcasted to other extensions.
```

Comment:
Please reword the 2nd sentence so that it reads as follows:

"This event is handled by the Core extension, and the new privacy status is broadcasted to other extensions."